docs/components/stepper/index.html

```diff
@@ -92,7 +90,7 @@ <h2 id="basic-stepper">Basic Stepper</h2>
                 <span class="beta-hxStepLabel">Account</span>
                 <span class="beta-hxStepValue">Acme Corp Inc.</span>
               </header>
-              <div class="hxBox hxMd">
+              <div class="hxBox hxMd beta-hxContainerStyle">
```

Comment:
We shouldn't need the extra class.  Please remove.

Reply:
Changes have been done, please review.